Pre-migration CQA task for the REST API title (#4937)
theforeman/foreman-documentation: guides/common/modules/proc_configuring-ssl-authentication.adoc

= Configuring SSL authentication

[role="_abstract"]

> Configure SSL authentication to enable secure API requests from remote servers or directly on {ProjectServer} by installing and trusting the CA certificate.

Reviewer: Just a note - this use of "enable" is OK :)

Reviewer 2: Is it relevant where the user installs the cert on? IMO no. I suggest to shorten it:

```suggestion
Configure SSL authentication to enable secure API requests to {Project} by installing and trusting the CA certificate.
```
